src/ActionModal/components/DelegationModal.vue

```diff
@@ -150,8 +150,8 @@ export default {
         )
         return {
           type: transaction.REDELEGATE,
-          validator_src_address: validatorSrc.operator_address,
-          validator_dst_address: this.validator.operator_address,
+          validatorSourceAddress: validatorSrc.operator_address,
```

Comment:
This var naming must match the `cosmos-api/src/messages.js` functions. It's not enforced at a code level yet.

Reply:
;D